src/Routes.js

```diff
@@ -18,7 +18,6 @@ import Program from "./organizations/Program";
 import Register from "./register/Register";
 import Login from "./login/Login";
 import ProtectedRoute from "./ProtectedRoute";
-import Navigation from "./Navigation";
```

Comment:
no need of removing  navbar from here as it will make it disappear from other existing pages.